test/integration/thruster.cc

```diff
@@ -285,14 +314,50 @@ void ThrusterTest::TestWorld(const std::string &_world,
       {
         EXPECT_NEAR(0.0, angVel.X(), _baseTol);
       }
+      else if (_namespace == "deadband")
+      {
+        continue;
+      }
       else
       {
-        EXPECT_NEAR(omega, angVel.X(), omegaTol) << i;
+        ASSERT_NEAR(omega, angVel.X(), omegaTol) << i;
       }
     }
     EXPECT_NEAR(0.0, angVel.Y(), _baseTol);
     EXPECT_NEAR(0.0, angVel.Z(), _baseTol);
   }
+
+  modelPoses.clear();
+  propellerAngVels.clear();
+  propellerLinVels.clear();
+  // Make sure that when the deadband is disabled
+  // commands below the deadband should create a movement
+  auto latest_pose = modelPoses.back();
+  msgs::Boolean db_msg;
+  if (_namespace == "deadband")
+  {
+    // disable the deadband
+    db_msg.set_data(false);
+    db_pub.Publish(db_msg);
+    // And we send a command that is still below the deadband threshold
+    msg.set_data(_deadband / 2.0);
+    pub.Publish(msg);
+    // Check movements
+    fixture.Server()->Run(true, 1000, false);
+    std::this_thread::sleep_for(std::chrono::milliseconds(100));
+    EXPECT_LT(0.1, modelPoses.back().Pos().X());
```

Comment:
A few comments of what these expectations mean would be helpful. We usually check velocity as opposed to position as that results in less flaky tests.

Reply:
That is a good point, I'll use the same equations for checking the propeller speed used earlier in the test.